src/app.js

```diff
@@ -45,7 +51,8 @@ const swaggerDefinition = {
 // options for the swagger docs
 const options = {
   definition: swaggerDefinition,
-  apis: ['./docs/**/*.yaml'],
+  // apis: ['./docs/**/*.yaml'],
+  apis: [`${__dirname}/routes/auth.js`],
```

Comment:
Why did you changes this?

Reply:
To point the swagger config to the routes file. But I'll try and write the swagger inside the yaml file or however that works